front/src/common/IntervalsDataViz/data.ts

```diff
@@ -673,8 +677,8 @@ export function entityDoUpdate<T extends EditorEntity>(entity: T, sourceLine: Li
       }
     });
     // eslint-disable-next-line dot-notation
-    newProps['length'] = getLineStringDistance(entity.geometry as LineString);
-
+    if (entity.properties.length === 0)
```

Comment:
Ok so I took a look to the code and I think it is more relevant to move this test line 668. If the entity is not initialized, we update the data, otherwise we don't.
You can also add a comment to explain that the modification of the linestring modifies the entity real properties only during initialization.

If you do that, you can remove the condition l680 and l366-368 too :)